### ISSUE TITLE
I have a problem after training is complete

### ISSUE THREAD

**Issue by qiao89757** [cvg/LightGlue#88]
Hello, thank you very much for your open source code. I am currently encountering a problem. When I train image registration at a large angle, the configuration file is as follows:
data:
name: homographies
data_dir: revisitop1m
train_size: 150000
val_size: 2000
batch_size: 128
num_workers: 14
homography:
difficulty: 0.7
max_angle: 180
photometric:
name: lg
After I trained and fine-tuned, I loaded the generated trained model into lightglue. Even if I ran the code with the same image, the effect of the generated image was as
![2023-11-14 10-01-25屏幕截图](https://github.com/cvg/LightGlue/assets/145572244/2b8152ad-c423-4a11-9966-679bb213c78f)

**Comment by Phil26AT** [comment]
Hey, sorry for the late reply. It would be great if you could elaborate your setup with more details. Did the training converge? What is the final loss? How did you load the weights? In your example, both images are identical, so even the pretrained LG should give very good matches.
